config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -265,6 +265,261 @@ presubmits:
       testgrid-alert-email: antonio.ojea.garcia@gmail.com, danwinship@redhat.com
       description: Uses kubetest to run e2e Conformance, SIG-Network tests against a cluster using nftables created with cluster/kube-up.sh
 
+  - name: pull-kubernetes-e2e-gce-kubedns-performance
+    cluster: k8s-infra-prow-build
+    branches:
+    # TODO(releng): Remove once repo default branch has been renamed
+    - master
+    - main
+    always_run: false
+    run_if_changed: '^(test/e2e/network/|cluster/addons/dns/kube-dns/)'
```

Comment:
remove run_if_changed, we are trying to avod it https://github.com/kubernetes/test-infra/pull/33958

Reply:
Adjusted the release instruction in https://github.com/kubernetes/dns/pull/664 and removed `run_if_changed` here.